### DISCUSSION TITLE
Osu!lazer Best Performance not Updated

### DISCUSSION THREAD

**Discussion by luma0724** [ppy/osu#24083] [Q&A] [answered]
I encounter an issue similar to #22863.
I have been playing a while today and many scores should be in my best performance as shown in the screenshots (thank you for the good game! I improve a lot in osu!mania)

My recent plays:
<img width="1117" alt="Screenshot 2023-06-29 at 1 20 20 PM" src="https://github.com/ppy/osu/assets/116613598/f2b4c2fb-8df3-4cfd-8ffa-a503fbb7a133">
<img width="1169" alt="Screenshot 2023-06-29 at 1 20 36 PM" src="https://github.com/ppy/osu/assets/116613598/3c92c6ba-0534-415b-9d4e-cf38ce57e376">

My best performance:
<img width="1143" alt="Screenshot 2023-06-29 at 1 20 58 PM" src="https://github.com/ppy/osu/assets/116613598/3cc2031d-998b-43a5-add9-6b9a87adb043">

However, it seems like my best performance did not update since yesterday. I understand the update usually need to take sometime but I have encountered a similar issue before as well (a few months ago, and my best performance was not updated for a long time until I updated my osu!lazer version). How do you solve the issue as in #22863? If similar things happen again, what can I do? :<

I'm using macOS Monterey 12.4, MacBook Air (M1, 2020) with osu!lazer 2023.6.21.0 (the latest by this time).

**Comment by peppy** [comment]
Could you please link your profile?

**Reply by Theighlin** [reply]
Not OP, but does the fact that recently submitted lazer scores are not showing up on leaderboards matter at all?

![image](https://github.com/ppy/osu/assets/53081750/ffa87ae3-6c5b-4e89-8d12-70f94dd72838)
All these beatmaps have no scores in them (lazer.ppy.sh) and other users in lazer chat reported this, is it intended to be this way right now?
I have one loved score that is not showing up where it should. https://lazer.ppy.sh/users/11577732

Sorry if unrelated.

**Reply by peppy** [reply, marked as answer]
Thanks, I've found a cause of failure (new bug in elasticsearch indexer) and resolved it. Population has already caught back up, so things should be back to normal now.

**Reply by luma0724** [reply]
Great! Things are back normal. Thanks!
